### PR TITLE
fix(MultilevelCache):multilevel cache not update question,

### PR DESCRIPTION
**Pull request by MapleWithoutWords**
# Description
use ```Masa.Contrib.Caching.Distributed.StackExchangeRedis``` and ```Masa.Contrib.Caching.MultilevelCache``` package. when local remove  a cache, server end not remove.

### CODE REVIEW

**Comment by doddgu** [comment]
@zhenlei520 PTAL

**Review by zhenlei520 — CHANGES_REQUESTED** [review]
The Remove and RemoveAsync methods need to add the method of removing the subscription channel, and add unit tests to ensure that it is valid when subscribing again

File: src/Contrib/Caching/Masa.Contrib.Caching.MultilevelCache.Tests/MultilevelCacheClientTest.cs [hunk 1]
```diff
@@ -90,6 +90,31 @@ public async Task TestGetAndSubscribeAsync()
...
+        await _multilevelCacheClient.RemoveAsync<string>(key);
+        subscribeChannelCount = ((List<string>)subscribeChannelsField!.GetValue(_multilevelCacheClient)!).Count;
+        Assert.AreEqual(subscribeChannelCount, 0);
+
+        await _multilevelCacheClient.GetAsync<string>(key, newVal =>
+        {
+            value = newVal;
+        });
+        subscribeChannelCount = ((List<string>)subscribeChannelsField!.GetValue(_multilevelCacheClient)!).Count;
+        Assert.AreEqual(subscribeChannelCount, 1);
```
Comment (zhenlei520) [inline src/Contrib/Caching/Masa.Contrib.Caching.MultilevelCache.Tests/MultilevelCacheClientTest.cs:115]: The expected value and the target value are wrongly written

```csharp
Assert.AreEqual(1, subscribeChannelCount);
```

Comment (zhenlei520) [reply src/Contrib/Caching/Masa.Contrib.Caching.MultilevelCache.Tests/MultilevelCacheClientTest.cs:115]: Missing unit tests for sync delete

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

File: src/Contrib/Caching/Masa.Contrib.Caching.MultilevelCache.Tests/MultilevelCacheClientTest.cs (hunk 1 above)
Comment (MapleWithoutWords) [reply src/Contrib/Caching/Masa.Contrib.Caching.MultilevelCache.Tests/MultilevelCacheClientTest.cs:115]: ok,already repaired

**Review by zhenlei520 — APPROVED** [review]
LGTM

**Pull request merged**